docs/index.md

```diff
@@ -68,38 +50,19 @@ A how-to guide:
 
 ## [Discussions](discussions.md)
 
-A discussion:
-
-* is understanding-oriented
-* explains
-* provides background and context
-
-*Analogy: an article on culinary social history*
-
 * [Why Roll?](discussions.md#why-roll)
 
 
 ## [Reference](reference.md)
 
-A reference guide:
-
-* is information-oriented
-* describes the machinery
-* is accurate and complete
-
-*Analogy: a reference encyclopaedia article*
-
 * [Core objects](reference.md#core-objects)
 * [Extensions](reference.md#extensions)
 * [Events](reference.md#events)
 
 
 ## [Changelog](changelog.md)
 
-A changelog:
-
-* is breaking-change-oriented
-* links to related issues
-* is concise
+* [Changelog](changelog.md)
```

Comment:
Redundant but given that the link is also a title it might be confusing. At least to me, not sure how to handle that…